### ISSUE TITLE
Add notes to readme/documentation on accessibility

### ISSUE THREAD

**Issue by jkjustjoshing** [amio/re-carousel#131]
Right now there's no note on how this library acts for a screen reader, keyboard user, or just a user that doesn't read very fast. These are important considerations for a UI component to consider. 

Please check out https://www.w3.org/WAI/tutorials/carousels/ for more information on things to consider.

2 specific issues I noticed with a quick glance at the docs:
1. The buttons in the [button example](https://github.com/amio/re-carousel/blob/master/src/buttons.js) aren't buttons! Not friendly to keyboard OR screen reader users
2. The link to the [keyboard navigation](https://github.com/amio/re-carousel/blob/master/src/keyboard-navigator) example from the README is broken. Keyboard navigation is an important thing for keyboard-only users, though make sure that the implementation follows W3 accessibility guidelines!

**Comment by amio** [comment]
@jkjustjoshing Thanks for pointing out! I'm planning a rewrite, it's a good opportunity to have accessibility support baked in from start.
